.github/workflows/CI.yml

```diff
@@ -439,12 +439,17 @@ jobs:
         name: "Docker Build keptn/${{ matrix.config.artifact }}"
         if: matrix.config.should-push-image == 'true' && ( matrix.config.should-run == 'true' || needs.prepare_ci_run.outputs.BUILD_EVERYTHING == 'true' )
         uses: docker/build-push-action@v2
+        env:
+          DATETIME: ${{ needs.prepare_ci_run.outputs.DATE }}${{ needs.prepare_ci_run.outputs.TIME }}
+          GIT_SHA: ${{ needs.prepare_ci_run.outputs.GIT_SHA }}
         with:
           context: ${{ matrix.config.working-dir }}
           tags: |
             keptndev/${{ matrix.config.artifact }}:${{ env.VERSION }}
             keptndev/${{ matrix.config.artifact }}:${{ env.VERSION }}.${{ env.DATETIME }}
           build-args: |
             version=${{ env.VERSION }}
+            buildTime=${{ env.DATETIME }}
```

Comment:
Did you try out what happens when using skaffold to run the distributor? Are the variables just empty then?

Reply:
i think it just contained the noninterpolated string(s) $buildTime or $gitSha.
I've made a commit to provide default values.